collect_app/src/main/java/org/odk/collect/android/utilities/AuditEventLogger.java

```diff
@@ -62,41 +62,40 @@ public void logEvent(AuditEvent.AuditEventType eventType, boolean writeImmediate
      */
     public void logEvent(AuditEvent.AuditEventType eventType, FormIndex formIndex,
                          boolean writeImmediatelyToDisk, String questionAnswer) {
-
-        // Ignore beginning of form and repeat events
-        if (eventType == AuditEvent.AuditEventType.BEGINNING_OF_FORM || eventType == AuditEvent.AuditEventType.REPEAT) {
+        if (!isAuditEnabled()
```

Comment:
I would factor out this list of conditions to a separate method.

Reply:
What would that method be called? What do you think the advantage would be? All I can think of in this case would be something like `isEventLoggingRequired` but that makes understanding the `logEvent` method harder because of the added level of indirection that provides no information on when logging is unnecessary.

Reply:
I was thinking about `shouldEventBeLogged` but a method with the same name already exists in this class...
advantage? If you can factor out a part of a method giving a meaningful name (to the new one) it makes the code more readable you know it maybe even better.

>that makes understanding the logEvent method harder because of the added level of indirection that provides no information on when logging is unnecessary.

if a method has a good name and (or) a comment I don't think it's less clear.

Reply:
What do you think of 2e61ad2?

Reply:
What does it have to do with our conversation? I think you wanted to add it somewhere else right?

Reply:
Hmm, looks like I copied the wrong hash. 82d5c98 is what I meant.

Reply:
Looks fine.